translations/en.json

```diff
@@ -179,6 +179,11 @@
     "chat-settings": "Chat settings",
     "chat-with-friends": "Chat privately with friends",
     "chats": "Chats",
+    "check-before-syncing":"Check before syncing",
```

Comment:
Other translations also have empty space after `:`, maybe we should add for consistency.